### ISSUE TITLE
<raw> tag not working inside <if condition>

### ISSUE THREAD

**Issue by kuddl** [maizzle/framework#193]
- Maizzle Version: 1.0.3
- Node.js Version: 12.16.3

Hi there.

I love the project, thanks a lot for all your work and effort!

I want to achive the following:

In development: 
`Hi, Jolly Roger.`
In Production: 
`Hi, {{username}}`

I thought I'll be clever and tried:
```
<if condition="page.env === 'production'">
 <raw>
     {{ username }}
</raw>
</if>
 <else>
     Jolly Roger
 </else>
```
This worked for development, but in production the output was:
`undefined`

-> So I guess the `<raw>` tag does not work in a `<condition>`

Thx

**Comment by cossssmin** [comment]
Hi, please update to the latest version (1.0.9) and test again, it should work.

**Comment by cossssmin** [comment]
Just checked and it actually doesn't work, will open an issue with [posthtml-expressions](https://github.com/posthtml/posthtml-expressions). Thanks for bringing this up!

**Comment by richardhj** [comment]
Hi!
`<raw>` is not working inside header.html, which is included via `<component src="src/components/header.html"></component>`. The result is the same as described above. Is it the same issue?

**Comment by cossssmin** [comment]
@richardhj no, that is a different issue, thanks for reporting it!

**Comment by cossssmin** [comment]
@kuddl should be fixed in https://github.com/posthtml/posthtml-expressions/pull/87, which is now available in Maizzle v1.2.0

Feel free to reopen if you continue experiencing this issue.
